README.md

```diff
@@ -44,278 +44,316 @@ Unrecommended ones can still be considered. The most important thing is to remai
+-  node 🟡 $... • {selector / DOM}
+-  not equal • 🟡 ne {binary operator}
+-  number • 🟢 num
 
 ### O
 
--  🟢 obj • object
--  🔴 ord • order {data science}
--  🟡 op • operation {binary operation}
--  🟢 opt • option / optional
--  🟢 os • operating system
--  🟢 out • output
+-  object • 🟢 obj
+-  object key • 🟡 key {only with **val**}
+-  operation • 🟡 op {binary operation}
+-  optional • 🟢 opt
+-  option • 🟢 opt
+-  operating system • 🟢 os
+-  order • 🔴 ord {data science}
+-  output • 🟢 out
 
 ### P
 
--  🟡 p • pointer {memory}
--  🟢 param • parameter
--  🟢 perf • performance
--  🟢 pic • picture
--  🟢 pos • position
--  🟢 pred • prediction
--  🟢 pref • preference
--  🟢 prev • previous
--  🟡 proc • process {thread}
--  🟢 prod • production
--  🟢 prof • profiler
--  🟢 ptr • pointer
--  🟢 pr • pull request
--  🟡 pwr • power {energy}
--  🟢 px • pixel
+-  parameter • 🟢 param
+-  performance • 🟢 perf
+-  picture • 🟢 pic
+-  pixel • 🟢 px
+-  pointer
+   -  🟡 p {memory}
+   -  🟢 ptr
+-  position • 🟢 pos
+-  prediction • 🟢 pred
+-  preference • 🟢 pref
+-  previous • 🟢 prev
+-  process • 🟡 proc {thread}
+-  production • 🟢 prod
+-  profiler • 🟢 prof
+-  power • 🟡 pwr {energy}
 
 ### Q
 
--  🟢 q / qry • query
+-  query
+   -  🟢 q
+   -  🟢 qry
 
 ### R
 
--  🟡 r • radius {circle / shape}
--  🟢 rand • random
--  🟢 rec • record
--  🟡 rect • rectangle {shape}
--  🟢 recv • receive
--  🟢 ref • reference
--  🟢 regex / rgx • regular expression
--  🟢 rem • remove
--  🟢 repo • repository
--  🟢 req • required / requested
--  🟢 res • response / result
--  🟢 ret • return
--  🟢 rev • revision
--  🟢 rng • range
+-  radius • 🟡 r {circle / shape}
+-  random • 🟢 rand
+-  range • 🟢 rng
+-  receive • 🟢 recv
+-  record • 🟢 rec
+-  rectangle • 🟡 rect {shape}
+-  reference • 🟢 ref
+-  regular expression
+   -  🟢 rgx
+   -  🟢 regex
+-  remote • 🟢 rem
+-  repository • 🟢 repo
+-  required • 🟢 req
+-  requested • 🟢 req
+-  result • 🟢 res
+-  responde • 🟢 res
+-  return • 🟢 ret
+-  revision • 🟢 rev
 
 ### S
 
--  🟡 s • signed as prefix {modifier}
--  🟡 sem • semaphore {conditional operator}
--  🟢 sel • selected / selection
--  🟢 sin • sine
--  🟢 seq • sequence
--  🟢 sqrt • square root
--  🟢 src • source
--  🟡 sw • software {computer science}
--  🟢 stat • statistic
--  🟢 std • standard
--  🟢 str • string
--  🟢 sum • addition
--  🟢 sync • synchronize
+-  signed as prefix • 🟡 s {modifier}
+-  semaphore • 🟡 sem {conditional operator}
+-  selection • 🟢 sel
+-  selected • 🟢 sel
+-  sine • 🟢 sin
+-  sequence • 🟢 seq
+-  square root • 🟢 sqrt
+-  source • 🟢 src
+-  software • 🟡 sw {computer science}
+-  statistic • 🟢 stat
+-  std • standard • 🟢 std
+-  string • 🟢 str
+-  synchronous • sync
+-  synchronize • 🟢 sync
 
 ### T
 
--  🟡 t • time / type (e.g. uint8_t) {type modifier}
--  🟢 temp • temperature
--  🟢 temp / tmp • temporary
--  🔴 tgl • toggle
--  🟢 tmr • timer
--  🟢 ts • timestamp
--  🔴 txt • text
+-  temperature • 🟢 temp
+-  temporary
+   - 🟢 tmp
+   - 🟢 temp
```

Comment:
If "temp" is for "temperature" we should use "tmp" here.

Reply:
I prefer "temp" for temporary and "T" for temperature as I said in the first comment.

Reply:
My only issue with "T" is that if naming conventions requires "all_lower_case" format then e.g. "living_room_t" will be hard to decode. Anyway, let's pick one, to finalize just the format and we can think about it the next iteration.